pages/templates/pages/election_tracking_page.html

```diff
@@ -132,7 +132,7 @@ <h3>No Matching Results Found</h3>
 
         {% include "partials/past_results.html" %}
 
-        <script src="//cdnjs.cloudflare.com/ajax/libs/list.js/1.2.0/list.min.js"></script>
+        <script src="//cdnjs.cloudflare.com/ajax/libs/list.js/1.5.0/list.min.js"></script>
```

Comment:
I'm surprised there are no other changes, because I think this is a significant upgrade. For example I think fuzzy search is not a separate plugin anymore as of this version:
http://listjs.com/docs/plugins/fuzzysearch/
https://github.com/javve/list.fuzzysearch.js#deprecated-since-listjs-150

Also I briefly attempted to use latest version in my local dev when I updated this recently, and I had errors, so that's why I kept version at `1.2.0`. So unless I was mistaken there are other errors that show up with just a simple upgrade and no refactors. Did you test this thoroughly?

Reply:
I did test everywhere list.js was used and no issues, but will look into removing fuzzy search external.